gems/laa_multi_step_forms/app/controllers/steps/base_step_controller.rb

```diff
@@ -16,6 +16,10 @@ def edit
     def update
       raise 'implement this action, if needed, in subclasses'
     end
+
+    def reload
+      raise 'implement this action, if needed, in subclasses'
```

Comment:
I'm a little bit unsure of what this does and why it has been added. :) 

might be useful to document it. :) 

gems/laa_multi_step_forms/docs/Steps.md